__tests__/melt-tokens.test.js

```diff
@@ -5,11 +5,11 @@ const walletId = 'stub_melt_tokens';
 
 // TODO: Add a fixture to melt token 01
 describe('melt-tokens api', () => {
-  beforeAll(async () => {
+  beforeEach(async () => {
```

Comment:
I was having some problems after the first test that executed the melt with success, the others were failing because the wallet didn't have tokens to melt.

Reply:
This is curious. The unit tests were supposed to have no side effects because of the lack of interaction with the blockchain.

Maybe it has to do with the new storage scheme building local indexes on the app?

This looks inoffensive to me, but I believe it's a flag that maybe we're having unwanted side effects from our methods.